src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -23,13 +25,19 @@ def format_simulant_id(data: pd.DataFrame) -> pd.Series:
     return data["random_seed"].astype(str) + "_" + data["simulant_id"].astype(str)
 
 
+def get_state_name(data: pd.DataFrame, metadate=None) -> pd.Series:
```

Comment:
typo: s/metadate/metadata

Reply:
Or wait, what is `metadate` supposed to be?

Reply:
Oops don't know how that got there.  Not even an arg that is needed.